OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitWebViewFragment.java

```diff
@@ -150,6 +155,13 @@ public void onPageLoadProgressChanged(WebView webView, int progress) {
         }
     }
 
+    private void evaluateXBlocksForBanner() {
+        List<BlockType> notAllowedBlocks = Arrays.asList(BlockType.DISCUSSION, BlockType.HTML, BlockType.VIDEO);
```

Comment:
IMO, please create a list of xblocks on which we are gonna show this banner

Reply:
Okay. :+1: